src/pydata_sphinx_theme/assets/scripts/pydata-sphinx-theme.js

```diff
@@ -212,18 +212,26 @@ var toggleSearchField = () => {
 
   // if the input field is the hidden one (the one associated with the
   // search button) then toggle the button state (to show/hide the field)
-  let searchPopupWrapper = document.querySelector(".search-button__wrapper");
-  let hiddenInput = searchPopupWrapper.querySelector("input");
+  let searchDialog = document.querySelector(".search-button__search-container");
```

Comment:
```suggestion
  const searchDialog = document.querySelector(".search-button__search-container");
```

Nitpick too, if you want me to nitpick. (I don't particularly care)

Is there a linter that could auto fix those ?